docs/UserGuide.md

```diff
@@ -2,41 +2,70 @@
 
 ## Introduction
 
-{Give a product intro}
+`MindMyMoney` (M<sup>3</sup>) was proposed for users with multiple payment methods who wish to track their expenditures. 
+Conducting anecdotal interviews, we realized users face difficulty tracking their expenditures across different payment 
+platforms on vastly different items. `MindMyMoney` was then conceptualized on the basis of tracking user’s expenditures.
```

Comment:
Hmm maybe these 2 lines not necessary? I feel the introduction of UG should just explain what the product is/what it does.